addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
+
+        <record id="edi_fatturaPA" model="account.edi">
+            <field name="name">FatturaPA</field>
+        </record>
```

Comment:
Very strange to have the declaration of a new record inside a `.+template.xml` file